docs/en/observability/profiling-no-kernel-version-check.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[no-kernel-version-check]]
+= Override Kernel Version Check 
+
+The `-no-kernel-version-check` configuration option is used to control the kernel version compatibility check during the Profiling host agent's startup process. The kernel version check is used to enforce the minimum kernel version supported, currently version 4.15, and to prevent running the profiling agent on certain kernel versions with known issues. When this option is set to `true`, the compatibility check is bypassed, allowing the process to proceed regardless of the kernel version. The default value for this option is `false`, which means that the kernel version check is performed as usual.
```

Comment:
```suggestion
The `-no-kernel-version-check` flag, or the `no-kernel-version-check` key in the host-agent configuration file, is used to control the kernel version compatibility check during the Profiling host agent's startup process. The kernel version check is used to enforce the minimum kernel version supported, currently version 4.15, and to prevent running the profiling agent on certain kernel versions with known issues. When this option is set to `true`, the compatibility check is bypassed, allowing host agent execution to proceed regardless of the kernel version. The default value for this option is `false`, which means that the kernel version check is performed as usual.
```